app/models/cfe/submission.rb

```diff
@@ -0,0 +1,10 @@
+module CFE
+  class Submission < ApplicationRecord
+    include CFESubmissionStateMachine
+
+    self.table_name = :cfe_submissions
```

Comment:
I think if you add `CFE` as an acronym, I don't think you'll need to specify the table name. See:

- https://stackoverflow.com/questions/28990466/rails-autoloading-fully-capitalized-name-like-api
- https://borgs.cybrilla.com/tils/handling-acronyms-with-inflections/